#goal definition
bool liftObject
---
#result definition
bool success
string reference_frame_id
sensor_msgs/PointCloud pointCloud
geometry_msgs/PoseStamped finger1Pose
geometry_msgs/PoseStamped finger2Pose
geometry_msgs/PoseStamped finger3Pose
geometry_msgs/PoseStamped gripperPose
---
#feedback
string currentStep
